src/models/index.js

```diff
@@ -0,0 +1,18 @@
+const CatModel = require('./cat');
+const Sequelize = require('sequelize');
+
+const setUpDatabase = () => {
+    const connection = new Sequelize("have_i_fed_the_cat_app_db", "root", "password", {
+        host: "localhost",
+        port: 3307,
+        dialect: "mysql"
+    })
+
+    const Cat = CatModel(connection, Sequelize);
+
+    connection.sync({alter: true});
```

Comment:
Dont really need all the extra blank lines here. Its good to give the return statement a clear line above it, the others can be compressed together: 
```
    const Cat = CatModel(connection, Sequelize);
    connection.sync({alter: true});
```